src/apps/Agreement/DisputableApps/TokenAmount.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
```

Comment:
This would be a more consistent interaction, though there are some style differences with the parens. I'll default to this and check in with @dizzypaty about the change.